src/marine/glider2ioda.py

```diff
@@ -65,29 +64,43 @@ def _read(self):
         errs = np.squeeze(errs)
         ncd.close()
         base_date = datetime(1970, 1, 1)
-        for i in range(len(time)-1):
-            for j in [0, 1]:
-                valKey = vName[j], self.writer.OvalName()
-                errKey = vName[j], self.writer.OerrName()
-                qcKey = vName[j], self.writer.OqcName()
-                dt = base_date + timedelta(seconds=int(time[i]))
-                locKey = lats[i], lons[i], dpth[i], dt.strftime(
-                    "%Y-%m-%dT%H:%M:%SZ")
-                if j == 0:
-                    self.data[0][locKey][valKey] = temperature[i]
-                    self.data[0][locKey][errKey] = errs[i]
-                    self.data[0][locKey][qcKey] = Tqcs[i]
-                else:
-                    self.data[0][locKey][valKey] = salinity[i]
-                    self.data[0][locKey][errKey] = errs[i]
-                    self.data[0][locKey][qcKey] = Sqcs[i]
+        dxy = 1.0
+        dz = 1.0
+        ii = int((98-8)/dxy)+10
+        jj = int((45-2)/dxy)+10
+        kk = int((1000)/dz)+10
+        box = np.zeros((ii, jj, kk))
+        for i in range(len(lons) - 1):
+            if lats[i] > 2.0 and lats[i] < 45.0 and lons[i] < -8.0 and lons[i] > -98.0:
```

Comment:
Can you add some comments regarding the lat/lon values? Might be better to have these values as parameters in the class, instead of hard-coding them.

Reply:
The numbers stands for the domain of North Atlantic Ocean HAT10 domain.
The title should have implied North Atlantic Summer Underwater Glider.

Reply:
Could you add them as parameters of the class, e.g. something like `self.lat_sth_HAT10 = 2.0` and `self.lat_nrth_HAT10 = 45.0`?

Reply:
Now that all thinning related parameters of the class have been added into : test/testinput/glider.yaml